FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -121,6 +150,70 @@ - (FIRTrace *)activeTrace {
   return self.backgroundSessionTrace;
 }
 
+/**
+ * Checks if prewarming is available for the platform on current device.
+ * It is available when running iOS 15 and above.
+ *
+ * @return true if the platform could prewarm apps on the current device
+ */
++ (BOOL)isPrewarmAvailable {
+  if (![[[GULAppEnvironmentUtil applePlatform] lowercaseString] isEqualToString:@"ios"]) {
+    return NO;
+  }
+  NSString *systemVersion = [GULAppEnvironmentUtil systemVersion];
+  if ([systemVersion length] == 0) {
+    return NO;
+  }
+  return [systemVersion compare:@"15" options:NSNumericSearch] != NSOrderedAscending;
+}
+
+/**
+ RC flag for dropping all app start events
+ */
+- (BOOL)isAppStartEnabled {
+  return [self.configurations prewarmDetectionMode] != PrewarmDetectionModeNone;
+}
+
+/**
+ RC flag for enabling prewarm-detection using ActivePrewarm environment variable
+ */
+- (BOOL)isActivePrewarmEnabled {
+  PrewarmDetectionMode mode = [self.configurations prewarmDetectionMode];
+  return (mode == PrewarmDetectionModeActivePrewarm ||
```

Comment:
Is ActivePrewarmEnabled when the mode is PrewarmDetectionModeAll?

Reply:
It depends on how we define "None" and "All". in the old naming, 0 = "DropAllEvents", 4 = "KeepAllEvents". If "None" means no events are allowed and "All" means all events are allowed, then it works. If "None" means prewarm-detection is not going to identify any prewarm and "All" means prewarm-detection is going to identify all events as prewarm, then it we need to swap the ENUM names or swap 0 and 4.

Reply:
Perfect. Makes sense. Can we make it clear in the code on this definition and align the code accordingly? This was not evident when I looked at the code. May be better comments in the enum definition or the comments above this method can help.

Reply:
Added comments to ENUM definitions. Changed "None" and "All" to "KeepNone" and "KeepAll" to be more clear.